src/dotty/tools/dotc/core/TypeErasure.scala

```diff
@@ -112,9 +112,10 @@ object TypeErasure {
   /** The standard erasure of a Scala type. Value classes are erased as normal classes.
    *
    *  @param tp            The type to erase.
-  */
-  def erasure(tp: Type)(implicit ctx: Context): Type =
-    erasureFn(isJava = false, semiEraseVCs = false, isConstructor = false, wildcardOK = false)(tp)(erasureCtx)
+   *  @param eraseOrType   Whether erase OrType to its lowest uppper bound
+   */
+  def erasure(tp: Type, eraseOrType: Boolean = true)(implicit ctx: Context): Type =
+    erasureFn(isJava = false, semiEraseVCs = false, isConstructor = false, wildcardOK = false)(tp, eraseOrType)(erasureCtx)
```

Comment:
I think it would be more consistent to add this parameter to the `TypeErasure` class like `semiEraseVCs`, etc.